projects/browser-extension/vue-app/src/views/apple-ios.view.vue

```diff
@@ -7,17 +7,12 @@
       </div>
       <properties-list class="apple-ios__properties-list" v-else>
         <properties-item
-          v-for="item in appleMetaData"
-          :key="item.keyName"
-          :key-name="item.keyName"
-          :refresh-on="appleMetaData"
+          v-for="(item, index) in appleMetaData"
+          :key="index"
```

Comment:
Voor zover ik het weet, is het onverstandig om de `index` als `key` te gebruiken, waarom gebruik je niet de `term`? Dit geld voor alle `v-for`'s.

Reply:
Done.